ntbs-service-unit-tests/DataMigration/ImportValidatorTest.cs

```diff
@@ -0,0 +1,166 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Moq;
+using ntbs_service.DataAccess;
+using ntbs_service.DataMigration;
+using ntbs_service.Models.Entities;
+using ntbs_service.Models.Enums;
+using ntbs_service.Models.Validations;
+using Xunit;
+
+namespace ntbs_service_unit_tests.DataMigration
+{
+    // This test suite is not designed to test that the validation is correct, only that the validation is happening on
+    // all parts of a notification that we expect it to happen on - which is why it isn't testing many validation cases.
```

Comment:
As mentioned here I added this test suite as a way to test that validation is happening, not that it's doing it perfectly.